packages/frontend-2/components/project/page/settings/collaborators/Collaborators.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <ProjectPageSettingsBlock title="Collaborators" :icon="UsersIcon">
+    <template #introduction>
+      <p>Invite new members or edit roles for existing ones.</p>
+    </template>
+    <template #topButtons>
```

Comment:
Slot names should be kebab-cased because of https://vuejs.org/style-guide/rules-strongly-recommended.html#prop-name-casing

(Slot names are specified similarly to props in the template)